src/marqo/tensor_search/tensor_search.py

```diff
@@ -2302,7 +2311,7 @@ def vectorise_multimodal_combination_field_unstructured(field: str,
                 sub_field_name_list.extend(text_field_names)
 
         if len(image_content_to_vectorise) > 0:
-            with RequestMetricsStore.for_request().time(f"create_vectors.image"):
+            with RequestMetricsStore.for_request().time(f"create_vectors"):
```

Comment:
Is this just going to add to this metric or overwrite it?

Reply:
This is my concern as well, but I followed the old pattern from here: 2.11.4 https://github.com/marqo-ai/marqo/blob/7ab68c2fbd717ab7625385e73a75d60d2c58320d/src/marqo/tensor_search/tensor_search.py#L2119

However, separate from the RequestMetricsStore, we do calculate the overall time taken to vectorise and add to the variable `combo_vectorise_time_to_add` , passed to `_add_documents_structured` or `_add_documents_unstructured`, which uses the value in

```
logger.debug(f"          add_documents vectorise: took {(total_vectorise_time):.3f}s for {batch_size} docs, "
                 f"for an average of {(total_vectorise_time / batch_size):.3f}s per doc.")```

Reply:
It will add it in a list